cunumeric/eager.py

```diff
@@ -383,6 +383,22 @@ def transpose(self, rhs, axes):
             else:
                 self.array[:] = np.transpose(rhs.array, axes)
 
+    def repeat(self, repeats, axis, scalar_repeats):
+        if not scalar_repeats:
+            self.check_eager_args(repeats)
+        if self.deferred is not None:
+            return self.deferred.repeat(
```

Comment:
Remove the `return` here. This statement doesn't return anything, it just modifies `self.deferred`.